### ISSUE TITLE
How to filter the ST-VQA data set?

### ISSUE THREAD

**Issue by mmsyan** [Yuliang-Liu/MultimodalOCR#12]
Thank you for your work. In Table 1 of the paper, it shows that the ST-VQA image data set is exactly 4000, but the number of images in the file test_task1\2\3.json I downloaded is greater than 4000. Could you tell me how to filter the ST-VQA data set?

**Comment by echo840** [comment]
Thank you for your attention. As the test set of STVQA does not provide ground truth answers, we performed testing on the training set. However, the training set is too large, so we only selected the first 5000 questions and corresponding images from the task3 JSON file as the test set.

**Comment by ProgrammerZhujinming** [comment]
Hi, I have a similar question that I would like to consult with you. I find that COCO、CTW and TotalText need to be croppep, but this will produce more samples than mentioned in the paper.  Besides, OCR-VQA  and DovVQA have the same problem. Thank you for your reply at your convenience.

**Comment by echo840** [comment]
Thanks for your attention. You can download text recognition datasets from the link in readme. And in VQA, KIE and HMER tasks, if a dataset contains far more than 5000 questions, we will test the first 5000 questions in annotation file.
